go.mod

```diff
@@ -12,19 +12,19 @@ require (
 	github.com/grpc-ecosystem/grpc-gateway v1.16.0
 	github.com/kylelemons/godebug v1.1.0
 	github.com/rakyll/statik v0.1.7
-	github.com/spf13/cast v1.4.1
-	github.com/spf13/cobra v1.4.0
-	github.com/stretchr/testify v1.7.1
+	github.com/spf13/cast v1.5.0
+	github.com/spf13/cobra v1.6.0
+	github.com/stretchr/testify v1.8.0
```

Comment:
This is the upgrade that is causing our CI to fail, is there a way to reference the older version of testify?

Reply:
We can downgrade them manually in the `go.mod` replace block. But what's failing exactly?

Reply:
Here's [an issue](https://github.com/stretchr/testify/issues/1199) pointing to what was failing. Great point tho, easily fixable as https://github.com/cosmos/interchain-security/pull/485/commits/dcd0b09febbeb4e10ebe12d7addd29c8b7b3d362